bin/templates/project/AndroidManifest.xml

```diff
@@ -31,8 +31,10 @@
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
     <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
     <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
 
-    <application android:icon="@drawable/icon" android:label="@string/app_name"
+    <application android:name="org.xwalk.core.XWalkApplication"
```

Comment:
In crosswalk-lite branch, I add an empty CordovaApplication instead here, seems could keep api more consistent.